lms/envs/aws.py

```diff
@@ -676,6 +676,12 @@
 
 PROGS_URLS = ENV_TOKENS.get('PROGS_URLS', PROGS_URLS)
 
+# Redirection options
+if FEATURES.get("EDRAAK_ENABLE_AUTH_EXTERNAL_REDIRECT"):
```

Comment:
```suggestion
```

That's an anti-pattern. Please avoid using it. I don't see any benefit of having settings variables appearing conditionally.

Reply:
Done